sharding/handler/smc_handler.py

```diff
@@ -220,15 +224,16 @@ def release_notary(self,
 
     def add_header(
             self,
```

Comment:
just thought if we can enforce this indentation throughout our codebase.
I prefer this style to the style
```
def release_notary(self,
```
Since it can reduce diff changes when there is a change in the method.

Reply:
👍 on this.
@jannikluhn @hwwhww ?

Reply:
I also like this style better, but I'd (slightly) prefer staying consistent with py-EVM and the rest of the Python+Ethereum ecosystem.